### DISCUSSION TITLE
Help needed for plotting convergence curves

### DISCUSSION THREAD

**Discussion by pallavsen007** [alchemistry/alchemlyb#332] [General]
Can such [plots](https://alchemlyb.readthedocs.io/en/latest/convergence.html) be plotted from BAR or FEP data?
If yes, can you provide me with a simple script to do so.

**Comment by xiki-tempula** [comment]
But the link that you have given has already given such a simple script?

**Comment by pallavsen007** [comment]
Isn't it from MBAR data? Does it spit out same plots using BAR data?

**Comment by xiki-tempula** [comment]
I see. In that case, you just change mbar to bar.
